sympy/vector/coordsysrect.py

```diff
@@ -98,6 +213,65 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
+        if transformation is None:
+            if parent is None:
+                if rotation_matrix is not None:
+                    rotation_equations = \
+                       lambda x, y, z: CoordSys3D._rotation_trans_equations(parent_orient, lambda x, y, z: (x, y, z))
+                else:
+                    rotation_equations = lambda x, y, z: (x, y, z)
+
+                # Information about location without parent doesn't have sense.
+                # See lines 80:99. Location is somehow treated only when parent is put.
+                # We are not able to create independent vector, so we can't assign
+                # anything to location.
+                translation_equations = lambda x, y, z: (x, y, z)
+
+                from sympy import symbols
+                x, y, z = symbols('x y z')
+                transformation = rotation_equations(*translation_equations(x, y, z))
+                lame_coefficients = _set_lame_coefficient_mapping('cartesian', (x, y, z))
+                inverse_transformation = None
```

Comment:
Well, this could be specified. You just have to immagine that there is some default coordinate system used as a reference if `parent == None`.